acceptance/Rakefile

```diff
@@ -0,0 +1,111 @@
+require 'rototiller'
+
+desc "Generate Beaker Host config"
+rototiller_task :host_config do |task|
+  task.add_env(name: 'BOLT_CONTROLLER', default: 'centos7-64')
+  task.add_env(name: 'BOLT_NODES',
+               default: 'ubuntu1604-64,osx1012-64,windows10ent-64')
+  ns = [ENV['BOLT_CONTROLLER'], ENV['BOLT_NODES']].join(',')
+  n  = ns.split(',')
+  n_new = []
+  n.each_with_index do |node, i|
+    roles = []
+    roles << 'bolt' if i.zero?
+    roles << if /win/ =~ node
+               'winrm'
+             else
+               'ssh'
+             end
+    node = node + roles.join(',') + '.'
+    n_new << node
+  end
+  nodes_final = n_new.join('-')
+  generate = "bundle exec beaker-hostgenerator"
+  generate += " #{nodes_final}"
+  generate += " > hosts.yaml"
+  sh generate
+  sh "cat hosts.yaml"
+end
+
+namespace :ci do
+  beaker_options = [
+    { name: '--hosts',
+      add_argument: {
+        name: 'hosts.yaml',
+        add_env: {
+          name: 'BEAKER_HOSTS_FILE',
+          message: 'Beaker hosts file'
+        }
+      } },
+    { name: '--tests',
+      add_argument: {
+        name: 'tests',
+        add_env: {
+          name: 'BEAKER_TESTS',
+          message: 'Beaker test path(s)'
+        }
+      } },
+    { name: '--preserve-hosts',
+      add_argument: {
+        name: 'never',
+        add_env: {
+          name: 'BEAKER_PREVERSE_HOSTS',
+          message: 'When should beaker keep hosts alive'
+        }
+      } }
+  ]
+  # The keyfile is separated so that it can be exclude if the value is not
+  # set. This is a rototiller bug QA-2761.
+  beaker_keyfile = {
+    name: '--keyfile',
+    add_argument: {
+      add_env: {
+        name: 'BEAKER_KEYFILE',
+        message: 'The private SSH key beaker will use to connect to SUTs'
+      }
+    }
+  }
```

Comment:
Can we run with `--debug` enabled by default? I like to see what commands are being executed, not just:

```
Install Bolt via git

  * Clone repo

  * Update submodules

  * Construct version based on SHA
    localhost $ scp /var/folders/8m/d6146zxd6bv_3h8y4ykvvxh40000gn/T/beaker20171101-62411-1a7ivt0 debian9-64-1:bolt/lib/bolt/version.rb {:ignore => }

  * Build gem

  * Install custom gem

  * Ensure install succeeded
```